portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,28 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random facts to the page.
  */
-function addRandomGreeting() {
-  const greetings =
+function addRandomFacts() {
+  const facts =
       ['\'Wait for it...legendary\'', '\'Talent wins games, but teamwork and intelligence wins championships\'', 
       'Fact: I love sports, I practiced Taekwondo for 10 years', '\'Suit up\'', 'Fact: I have 3 dogs'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const random = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  document.getElementById('random-fact-container').innerText = random;
+}
+
+/** Request content from server and add it to page */
+function getContentUsingArrowFunc() {
+  try{
+      fetch('/data')
+  .then(response => response.json())
+  .then((fruits) => document.getElementById('fetch-container').innerHTML = fruits);
+  }
+  catch(error){
```

Comment:
I don't really feel strongly about this, but you could just do:
fetch('/data')
.then(...)
.then(...)
.catch(error => console.error(...))

which is easier to read